sfge/src/test/java/com/salesforce/rules/unusedmethod/ConstructorsTest.java

```diff
@@ -145,16 +143,16 @@ public void constructorCalledViaImplicitSubclassConstructor_expectNoViolation(
      */
     @ValueSource(ints = {0, 1})
     @ParameterizedTest(name = "{displayName}: arity {0}")
-    @Disabled // TODO: FIX AND ENABLE THIS TEST
     public void constructorImplicitlyCalledByGrandchild_expectNoViolation(Integer arity) {
         // spotless:off
         String[] sourceCodes = new String[]{
             "global virtual class ParentClass {\n"
-            // Declare a constructor with the expected arity that does nothing in particular.
-          + "    protected ParentClass(" + StringUtils.repeat("boolean b", arity) + ") {}\n"
+            // Declare a constructor with 0-arity that does nothing in particular.
+          + "    protected ParentClass() {}\n"
```

Comment:
This was an error in the test. The `ChildClass` constructor will implicitly call the default (0-arity) constructor for the parent class, which is a problem if that constructor doesn't actually exist.